tracer_test.go

```diff
@@ -597,17 +597,12 @@ func TestTracerDefaultTransport(t *testing.T) {
 		require.Error(t, err)
 		assert.EqualError(t, err, "failed to parse ELASTIC_APM_SERVER_TIMEOUT: invalid duration never")
 
-		// Implicitly created Tracers will have a discard tracer.
+
+		// Implicitly created Tracers will have Inactive tracer incase of invalid configuration
 		apm.SetDefaultTracer(nil)
 		tracer = apm.DefaultTracer()
-
-		tracer.StartTransaction("name", "type").End()
-		tracer.Flush(nil)
-		assert.Equal(t, apm.TracerStats{
-			Errors: apm.TracerStatsErrors{
-				SendStream: 1,
-			},
-		}, tracer.Stats())
+        isActiveTracer :=tracer.Active()
+		assert.Equal(t,false,isActiveTracer)
```

Comment:
Why the variable? Also, Go's linter require spaces here.

```suggestion
		assert.Equal(t, false, tracer.Active())
```

Reply:
resolved! removed unwanted variable